docs/sources/flow/reference/components/prometheus.exporter.gcp.md

```diff
@@ -0,0 +1,157 @@
+| `metrics_prefixes`        | `list(string)` | One or more values from the supported [GCP Metrics](https://cloud.google.com/monitoring/api/metrics_gcp). These can be as targeted or loose as needed.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                               |         | yes      |
+| `extra_filters`           | `list(string)` | Used to further refine the resources you would like to collect metrics from. Please note that any string value within a particular filter string must be enclosed in escaped double-quotes. The structure for these filters is `<targeted_metric_prefix>:<filter_query>`. The `targeted_metric_prefix` is used to ensure the filter is only applied to the metric_prefix(es) where it makes sense. It does not explicitly have to match a value from `metric_prefixes`, but the `targeted_metric_prefix` must be at least a prefix to one or more `metric_prefixes`. The `filter_query` is applied to a final metrics API query when querying for metric data. The final query sent to the metrics API already includes filters for project and metric type. Each applicable `filter_query` is appended to the query with an AND. You can read more about the metric API filter options in [GCPs documentation](https://cloud.google.com/monitoring/api/v3/filters). | `[]`    | no       |
```

Comment:
We prefer to keep the description as short as possible. If the argument needs further explanation, it can be done at the bottom of the argument's table. See this https://grafana.com/docs/agent/latest/flow/reference/components/prometheus.exporter.unix/#arguments as example

Reply:
thanks! took a stab at moving all of the non-essential description to below the argument's table